torch/csrc/jit/runtime/operator.cpp

```diff
@@ -350,7 +351,7 @@ bool aliasAnalysisHasSpecialCaseFor(Symbol symbol) {
 void registerOperator(Operator&& op) {
   if (op.schema().is_varret()) {
     Symbol s = Symbol::fromQualString(op.schema().name());
-    if (!printerHasSpecialCaseFor(s)) {
+    if (!printerHasSpecialCaseFor(s) && !IsFusionSubgraphKind(s)) {
```

Comment:
nit: I think this is unnecessary because it's already checked in printerHasSpecialCaseFor, right?